python-nectar.spec

```diff
@@ -26,24 +27,43 @@ such as the default "threaded" downloader, which uses the "requests" library
 with multiple threads. Other experimental downloaders have used tools like
 pycurl and eventlets.
 
+%package -n python2-nectar
+Summary:        %{sum}
+Requires:       python-isodate >= 0.4.9
+Requires:       python-requests >= 2.4.3
+%description -n python2-nectar
+
```

Comment:
Same for description which can have global definition and %{desc} macro in main package and all subpackages